### PR TITLE
merge fix

### PR DESCRIPTION
**Pull request by KyryloYakymchuk**

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -1,4 +1,15 @@
 table = document.getElementById('table');
+const buttonList = document.getElementsByClassName('deleteButton')
+const cookieValue = document.getElementsByClassName('cookieValue')
+const cookieName = document.getElementsByClassName('cookieName')
+const nameInput = document.getElementById('nameInput')
+const valueInput = document.getElementById('valueInput')
+const valueDate = document.getElementById('valueDate')
+const createButton = document.getElementById('createButton')
```
Comment (Andrew01011997) [inline popup.js]: save here only object with id`s

File: popup.js [hunk 2]
```diff
@@ -23,96 +34,92 @@ const drawHTML = (elem, cookies) => {
       <textarea cookieId=${index} class='cookieName'>${name}</textarea>
       <textarea cookieId=${index} class='cookieValue'>${value} </textarea>
      </div>
-     <span class='cookieLiveTime'>${(new Date(expirationDate * 1000)).toLocaleString()}</span>
+     <span class='cookieLiveTime'>${(new Date(expirationDate)).toLocaleString()}</span>
```
Comment (Andrew01011997) [inline popup.js:36]: why you change it?

File: popup.js [hunk 3]
```diff
@@ -23,96 +34,92 @@ const drawHTML = (elem, cookies) => {
...
        <button value=${index} class='deleteButton'>X</button>
     </div>
   ` 
 ).join(' ')
 : elem.innerHTML = `<div class='noContent'>No cookies!</div>`
 }
 
+const updateCookie = (cookieData) =>{
+  console.log(cookieData);
```
Comment (Andrew01011997) [inline popup.js]: remove console.log

File: popup.js [hunk 4]
```diff
@@ -23,96 +34,92 @@ const drawHTML = (elem, cookies) => {
...
+  chrome.cookies.set({
+    ...cookieData
```
Comment (Andrew01011997) [inline popup.js]: no sense of destructing here

File: popup.js [hunk 5]
```diff
@@ -23,96 +34,92 @@ const drawHTML = (elem, cookies) => {
...
+          const {name, storeId, secure, domain, path, expirationDate} = cookies[Number(e.target?.attributes?.[0]?.value)]
+          const protocol = secure ? "https:" : "http:";
+          const cookieUrl = `${protocol}//${domain}${path}`;
           chrome.cookies.remove({
             url: cookieUrl,
-            name: cookie.name,
-            storeId: cookie.storeId
+            name: name,
+            storeId: storeId
           })
```
Comment (Andrew01011997) [inline popup.js:86]: move this logic to update cookie function

**Review by Andrew01011997 — CHANGES_REQUESTED** [review]

File: popup.js (hunk 2 above)
Comment (KyryloYakymchuk) [reply popup.js:36]: Tam eto nenado

File: popup.js [hunk 6]
```diff
@@ -59,22 +76,20 @@ chrome.tabs.query({ currentWindow: true, active: true }, function (tabs) {
...
-            url:cookieUrl,
-            name:cookie.name,
-            value:e.target.value,
-            storeId: cookie.storeId
+          updateCookie({
+            url: cookieUrl,
+            value: e.target.value,
+            name,
+            storeId,
+            expirationDate
```
Comment (Andrew01011997) [inline popup.js:92]: image
Add ALL this logic to updateCookie function

**Review by Andrew01011997 — CHANGES_REQUESTED** [review]

**Pull request merged**